sql/2019/05_Third_Parties/05_06.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# Top 100 third party domains by request volume
+SELECT
+  thirdPartyDomain,
+  COUNT(*) AS totalRequests,
+  SUM(requestBytes) AS totalBytes
+FROM (
+  SELECT
+      SAFE_CAST(REGEXP_EXTRACT(payload, r'_bytesIn":(\d+)') AS INT64) AS requestBytes,
```

Comment:
If all you need is the response size, let's use `respSize` from the much smaller `summary_requests` dataset.

Reply:
Ah yeah everything got removed here I guess this is just response size at this point thanks! :)